boards/phytec/phyboard_lyra/doc/phyboard_lyra_am62xx_m4.rst

```diff
@@ -137,7 +137,7 @@ Debugging
 The board is equipped with an XDS110 JTAG debugger. To debug a binary, utilize the `debug` build target:
 
 .. zephyr-app-commands::
-   :zephyr-app: <my_app>
+   :app: <my_app>
```

Comment:
I would assume this is actually incorrect and should be fixed, right?

Reply:
Yes and no, I don't really mind people using this as a way to indicate how the build comment would *generally* look like https://builds.zephyrproject.io/zephyr/pr/78064/docs/boards/phytec/phyboard_lyra/doc/phyboard_lyra_am62xx_m4.html#debugging

Reply:
Or if what you mean is that I could have stuck to `:zephyr-app:`, yes, that's true, but the intent was to indicate that this can be *any* app, doesn't have to be in-tree

Reply:
No, I meant if this should use a specific sample but as you're pointed out, it makes sense in the documentation. Thanks for clarification.